waku/v2/node/storage/message/waku_message_store.nim

```diff
@@ -12,18 +13,59 @@ import
 
 export sqlite
 
+logScope:
+  topics = "wakuMessageStore"
+
 const TABLE_TITLE = "Message"
+const maxStoreOverflow = 1.3
```

Comment:
I suggest using capital case instead (it is easier to distinguish local vars from const)
```suggestion
const MAX_STORE_OVERFLOW = 1.3
```

Reply:
I did not use capital case because of [NEP1](https://nim-lang.org/docs/nep1.html#introduction-naming-conventions), and I did not find anything overriding this in [status style guide](https://status-im.github.io/nim-style-guide/00_introduction.html).
No strong opinion on my side. Wdyt?

Reply:
Yeah, no strong opinions either, other than that we've certainly not been consistent so far. :/ I suggest we go with the linked Nim convention and specifically PascalCase - this is also what `nim-libp2p` uses. A good exercise for our next focused refactoring effort. Thanks for highlighting, @staheri14.